cde-core/resource/js/cdf-dd-components-selectors.js

```diff
@@ -31,10 +31,10 @@ var ListenersRenderer = SelectMultiRenderer.extend({
     var data = Panel.getPanel(ComponentsPanel.MAIN_PANEL).getParameters();
     var _str = "{";
     $.each(data, function(i, val) {
-      _str += "'" + val.properties[0].value + "': '" + val.properties[0].value + "',";
+      _str += '"' + val.properties[0].value + '": "' + val.properties[0].value + '",';
     });
 
-    _str += " 'selected':" + (this.value.replace(/\$\{p:(.+?)\}/g, '$1')) + "}";
+    _str += ' "selected":' + (this.value.replace(/\$\{p:(.+?)\}/g, '$1')) + "}";
```

Comment:
_str += ' "selected":' + (this.value.replace(/\$\{p:(.+?)\}/g, '$1')**.replace(/\'/g, '"')**) + "}";